application/open-webui/README.md

```diff
@@ -0,0 +1,64 @@
+# Use CrateDB with Open WebUI
+
+## About
+
+A complete end-to-end rig including CrateDB, CrateDB MCPO, and Open WebUI. 
+
+[Open WebUI] is an extensible, feature-rich, and user-friendly self-hosted AI
+platform designed to operate entirely offline. It supports various LLM runners
+like Ollama and OpenAI-compatible APIs, with built-in inference engine for RAG,
+making it a powerful AI deployment solution.
+
+CrateDB MCPO is a wrapper around CrateDB MCP, publishing the exposed tools via
+OpenAPI, to connect it to Open WebUI.
```

Comment:
The section could be slightly more elaborate, including references to **all** included artefacts.